Dockerfile

```diff
@@ -95,6 +101,9 @@ conda clean -afy
 pip cache purge
 EOF
 
+# Disable the JupyterLab announcements
+RUN /opt/conda/bin/jupyter labextension disable "@jyupyterlab/apputils-extension:announcements"
```

Comment:
Is this a typo?
```suggestion
RUN /opt/conda/bin/jupyter labextension disable "@jupyterlab/apputils-extension:announcements"
```

Reply:
Oof, copy-paste error. And bit disappointed this did not fail the build.